woocommerce/woocommerce-android
# [UOE] Update order sync failed event

Part of: #6711

### Description
This PR adds the `flow` parameter to order sync failed event:

- order_sync_failed

### Testing instructions
TC1

1. Go to Orders
2. Select an Order -> press Edit
3. Turn `Plain mode` ON in the device
4. Made a change in the order
5. Check that `🔵 Tracked: order_sync_failed, Properties: {"flow":"editing",...}` is triggered

TC2

1. Go to Orders
2. Tap (+) Add Button -> Create order
3. Turn `Plain mode` ON in the device
4. Add a product to the order
5. Check that `🔵 Tracked: order_sync_failed, Properties: {"flow":"editing",...}` is triggered
